executor/analyze.go

```diff
@@ -962,47 +982,30 @@ func (e *AnalyzeFastExec) handleSampTasks(bo *tikv.Backoffer, workID int, err *e
 	if *err != nil {
 		return
 	}
+	snapshot.SetOption(kv.NotFillCache, true)
+	snapshot.SetOption(kv.IsolationLevel, kv.RC)
+	snapshot.SetOption(kv.Priority, kv.PriorityLow)
```

Comment:
Why we need these?

Reply:
These are optimizations.

`NotFillCache` & `PriorityLow`: the sampling task should not affect the normal kv operations.
`Read Committed Isolation`: it speeds up the kv scan operation.

Reply:
OK.